__tests__/app.test.js

```diff
@@ -1,46 +1,83 @@
+/** * @jest-environment node */
 const request = require('supertest');
-const app = require('../src/app');
 const nock = require('nock');
+const app = require('../src/app');
+const { mockListOfJokes, mockRandomJokes, mockPersonalJokes } = require('../data/test-data');
 
-describe('My jokes API', () => {
-  describe('API GET /', () => {
-    it('should respond with a welcome message', done => {
-      request(app)
-        .get('/')
-        .then(res => {
-          expect(res.statusCode).toEqual(200);
-          expect(res.body.message).toEqual('Welcome to my jokes API!');
-          done();
-        });
-    });
-  });
-  it('that this is the jokes endpoint', done => {
+describe('GET / - Homepage', () => {
+  it('should respond with some homepage markup', async () => {
     request(app)
-      .get('/jokes')
+      .get('/')
       .then(res => {
         expect(res.statusCode).toEqual(200);
-        expect(res.body.jokes).toEqual(
-          'You have reached the jokes endpoint, please laugh with care',
-        );
-        done();
+        expect(res.text).toContain('Hello, Welcome to my jokes API');
       });
   });
-  it('should respond with a random joke message', done => {
-    request(app)
+});
+
+describe('GET /jokes', () => {
+  it('that jokes endpoint responds with list of jokes', async () => {
```

Comment:
"it that jokes endpoints.." I would remove the that and replace with something like 'it returns a list of jokes'. We already know which endpoint you are referencing because the describe block above tells us. Also the standard for it blocks is that the name of the test should read from the it. Eg. "it should do something", "it should return something" etc